name-computer/map.jinja

```diff
@@ -23,6 +26,13 @@ RedHat:
       - hostname: {{ name }}
       - apply_hostname: True
       - retain_settings: True
+
+  Update /etc/hosts:
+    host.present:
+      - ip: {{ salt['network.interface'](salt['network.default_route']('inet')[0]['interface'])[0]['address'] }}
```

Comment:
Use the dotted notation for salt commands... `salt.network.interface(...)`...

Reply:
Addressed in e2fab6df